README.md

```diff
@@ -5,7 +5,65 @@ This repository contains the code to generate various versions of the Datadog Ag
+|`DD_UPGRADE`|Set to any value to trigger a version upgrade from datadog-agent 5. It will import configuration files from `/etc/dd-agent/datadog.conf`. Not compatible with `DD_AGENT_FLAVOR=datadog-dogstatsd`.|
+|`DD_FIPS_MODE`|Set to any value to enable the use of the FIPS proxy to send data to the DataDog backend. Enabling this will force all outgoing traffic from the Agent to the local proxy. It's important to note that enabling this will not make the Datadog Agent FIPS compliant, but will force all outgoing traffic to a local FIPS compliant proxy. By-pass `DD_SITE` and `DD_URL` when enabled|
+|`DD_ENV`|The environment name where the agent is running. Attached in-app to every metric, event, log, trace, and service check emitted by this Agent.|
+|`DD_APM_INSTRUMENTATION_ENABLED`|Automatically instrument all your application processes alongside agent installation. Possible values are `host`, `docker` or `all` (`all` is both `host` and `docker`). `docker` value will check if docker is installed and will set `DD_NO_INSTALL` to `true`. If `DD_APM_INSTRUMENTATION_LANGUAGES` is not set, it will enable all libraries: `java`, `js`, `python`, `dotnet` and `ruby`. `host` injection is not compatible with the `DD_NO_AGENT_INSTALL` option. Only supported on Agent v7. Not supported on SUSE.|
+|`DD_APM_INSTRUMENTATION_LANGUAGES`|Specify the APM library to be installed. Possible values are `all`, `java`, `js`, `python`, `dotnet` and `ruby`. Does not run the APM injection script, which are triggered by `DD_APM_INSTRUMENTATION_ENABLED`.|
+|`DD_APM_INSTRUMENTATION_NO_CONFIG_CHANGE`|Set to any value to pass the `--no-config-change` option to the APM host injection script.|
+|`DD_SYSTEM_PROBE_ENSURE_CONFIG`|Create the system probe configuration file from templace, if it does not already exist.|
+|`DD_RUNTIME_SECURITY_CONFIG_ENABLED`|If set to `true`, ensure creation of security agent and system probe configuration file (if they don't already exist), and enable Cloud Workload Security (CWS).|
+|`DD_COMPLIANCE_CONFIG_ENABLED`|If set to `true`, ensure creation of security agent configuration file (if it doesn't already exist), and enable Cloud Security Posture Management (CSPM).|
+|`DD_INSTALL_ONLY`|Set to any value to prevent starting the agent after installation.|
+|`DD_NO_AGENT_INSTALL`|Do not install the agent. It will install package signature keys and create configuration files if they don't already exist.Automatically set to `true` when `DD_APM_INSTRUMENTATION_ENABLED=docker`.|
+
+
+## Install script configuration options
+Install script also comes with its own configuration options for testing purpose.
+
+>[!WARNING]
+> These options are for datadog internal use only.
+
+| Variable | Description|
+|-|-|
+|`DD_INSTRUMENTATION_TELEMETRY_ENABLED`|`true` if not set. When `false`, won't report telemetry to Datadog in case of script issue.|
```

Comment:
```suggestion
|`DD_INSTRUMENTATION_TELEMETRY_ENABLED`|`true` if not set. When `false`, doesn't report telemetry to Datadog in case of a script issue.|
```